Akinator.Api.Net/Utils/ServerSelector.cs

```diff
@@ -46,10 +46,18 @@ public static class ServerSelector
                         new Server(ServerType.Animal, "srv13.akinator.com:9257/ws")
                     }
                 },
+                {
+                    Language.France, new[]
+                    {
+                        new Server(ServerType.Person, "srv3.akinator.com:9217/ws"),
+                        new Server(ServerType.Object, "srv3.akinator.com:9218/ws"),
+                        new Server(ServerType.Animal, "srv3.akinator.com:9259/ws")
+                    }
+                },
                 {
                     Language.Russian, new[]
                     {
-                        new Server(ServerType.Person, "srv14.akinator.com:9286/ws")
+                        new Server(ServerType.Person, "srv12.akinator.com:9190/ws")
```

Comment:
why did you change this?

Reply:
I changed this because the previous servers go down a lot of times and return KO - Server Down